soccer/gameplay/GameplayModule.hpp

```diff
@@ -101,6 +101,10 @@ class GameplayModule {
 
     void calculateFieldObstacles();
 
+    bool hasInsetChanged();
```

Comment:
Mark this as `const` since it doesn't modify the `GameplayModule` object that it's called on.